requirements-dev.txt

```diff
@@ -9,3 +9,5 @@ sqlparse==0.1.19 \
 django-extensions==1.6.7 \
     --hash=sha256:7c33f1be6acf7414539cfdef2967bf6240d3c9317551b5d84622dff7824cd3c6 \
     --hash=sha256:ebdfc329207231f63b5ccb591407aadbc1a524af752e52690f0646eb7ff52943
+django-sslserver==0.19 \
+    --hash=sha256:1363835229a0585f89c42f3beca836572f3f6babdc1508f13352aed84b0924b3
```

Comment:
You're right. SSL was useful during the development and it's not required on production.